### ISSUE TITLE
💡 [FEATURE] - Make a Weather Data Analysis Project

### ISSUE THREAD

**Issue by Abhay182005dat** [jfmartinz/ResourceHub#391]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

Greetings !!

I would like to make a Weather Data analysis Project (Beginner) to the Data Science folder.I hope that will help many and will also be a good project to learn from.

Thank You

### Explain your solution

So basically i would be making the project in the collab notebook and would conduct Data Analysis on weather Data by Datasets from Kaggle or any other open weather Datasets. :)

**Comment by Abhay182005dat** [comment]
Sir I have added my project on the Projects list .
Thank You again for giving me this chance so much grateful . =D

Thank You
